components/lvgl_ili9341/ili9341.c

```diff
@@ -159,6 +166,37 @@ void ili9341_enable_backlight(bool backlight)
 #endif
 }
 
+/**
+ * Rotate the display by 0, 90, 180 or 270 degrees.
+ * @param driver pointer to lvgl driver variable
+ * @param direction 0: 0, 1: 90, 2: 180, 3: 270 degrees
+ */
+void ili9341_rotate(lv_disp_drv_t * drv, uint8_t direction) {
+	uint8_t data[1];
+	switch(direction) {
+		/* 0 degrees: no rotation */
+		case 0: data[0] = (ILI9341_MAD_MV | ILI9341_MAD_BGR);
```

Comment:
These command needs to be different if `ILI9341_INVERT_DISPLAY` is enabled. I think you can OR with a `#define TFT_INVON   0x21`. You should be able to see that invert objects on the display even if your device looks correct with out it.

Reply:
Just left my office for xmas holidays, I will catch up on this in the new year. From your response I'm reading you would be willing to merge this feature into the master branch, but want it to work for the inverted display option too? I'll give it a try!